src/Particular.CodeRules/ConfigureAwaitAnalyzer/ConfigureAwaitCodeFix.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.CodeAnalysis.CodeFixes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CodeActions;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace Particular.CodeRules.ConfigureAwaitAnalyzer
+{
+    [ExportCodeFixProvider(ConfigureAwaitAnalyzer.RuleIdentifier, LanguageNames.CSharp)]
+    public class ConfigureAwaitCodeFix : CodeFixProvider
+    {
+        public override ImmutableArray<string> FixableDiagnosticIds
+        {
+            get
+            {
+                return ImmutableArray.Create(ConfigureAwaitAnalyzer.RuleIdentifier);
+
+            }
+        }
+
+        public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+            var diagnostic = context.Diagnostics.First();
+            var diagnosticSpan = diagnostic.Location.SourceSpan;
+
+            var declaration = root.FindToken(diagnosticSpan.Start)
+                .Parent
+                .FirstAncestorOrSelf<AwaitExpressionSyntax>();
+
+            context.RegisterCodeFix(CodeAction.Create("Add ConfigureAwait(false)", cancellationToken => AddConfigureAwait(context.Document, declaration, false, cancellationToken), "Add ConfigureAwait(false)"), diagnostic);
+            context.RegisterCodeFix(CodeAction.Create("Add ConfigureAwait(true)", cancellationToken => AddConfigureAwait(context.Document, declaration, true, cancellationToken), "Add ConfigureAwait(true)"), diagnostic);
```

Comment:
Almost never makes sense for us to use ConfigureAwait(false). Since these are our code rules would it make sense to leave this option out?